mmhuman3d/core/evaluation/eval_utils.py

```diff
@@ -197,3 +217,172 @@ def keypoint_3d_auc(pred, gt, mask, alignment='none'):
     auc = pck_values.mean() * 100
 
     return auc
+
+
+def procrustes(X, Y, scaling=True, reflection='best'):
```

Comment:
We should have function that performs `procrustes`, see [here](https://github.com/open-mmlab/mmhuman3d/blob/21a8ce76a0e581e514f31cf31ff910bbb2d417be/mmhuman3d/core/evaluation/mesh_eval.py#L9). We should keep only one.

Reply:
changes updated